src/openforms/js/components/admin/forms/JsonWidget.js

```diff
@@ -5,13 +5,18 @@ import {useIntl} from 'react-intl';
 
 import {TextArea} from './Inputs';
 
+// dump JSON in a readable form
+const jsonFormat = value => {
+  return JSON.stringify(value, null, 2);
```

Comment:
Nice that it is formatted now :slightly_smiling_face:

Reply:
it's still not great but at least I can read it a bit now :P